tests/test_api.py

```diff
@@ -803,10 +807,10 @@ class MockParser(object):
             .with_args(TEST_GIT_URI, TEST_GIT_REF, git_branch=TEST_GIT_BRANCH, depth=None)
             .and_return(self.mock_repo_info()))
         flexmock(OSBS, _create_build_config_and_build=request_as_response)
-        req = osbs.create_prod_build(TEST_GIT_URI, TEST_GIT_REF,
-                                     TEST_GIT_BRANCH, TEST_USER,
-                                     TEST_COMPONENT, TEST_TARGET,
-                                     TEST_ARCH)
+        req = osbs.create_build(component=TEST_COMPONENT,
```

Comment:
maybe we should actually try to pass different component than is in Dockerfile, since we can't tell now from where was that req.spec.component.value set

Reply:
Nice catch, the test was actually completely broken.